<?xml version="1.0"?>
<launch>
    <node pkg="mdr_push_pull_object_action" type="push_pull_object_action" name="push_pull_object_server" output="screen">
        <param name="gripper_controller_pkg_name" value="mdr_gripper_controller" />
        <param name="move_arm_server" value="move_arm_server" />
        <param name="cmd_vel_topic" value="/cmd_vel" />
        <param name="movement_speed_ms" value="0.1" />
        <param name="safe_arm_joint_config" value="folded" />
        <param name="number_of_retries" value="0" />
    </node>
</launch>
